pkg/kubelet/dockertools/docker.go

```diff
@@ -625,4 +729,6 @@ func parseImageName(image string) (string, string) {
 
 type ContainerCommandRunner interface {
 	RunInContainer(containerID string, cmd []string) ([]byte, error)
+	ExecInContainer(containerID string, cmd []string, in io.Reader, out, err io.WriteCloser, tty bool) error
+	PortForward(containerID string, port uint16, stream io.ReadWriteCloser) error
```

Comment:
Should I move this to a new ContainerPortForwarder interface and add a portForwarder field to the Kubelet?